index.js

```diff
@@ -9,6 +9,7 @@ const instance = new LPJsonPollock();
 
 const init = instance.init.bind(instance);
 const render = instance.render.bind(instance);
+const validate = instance.render.bind(instance);
```

Comment:
is it an overlook?

change to `instance.validation.bind(instance)`

Reply:
Right!